packages/webapp/pages/onboarding.tsx

```diff
@@ -375,16 +378,7 @@ export function OnboardPage(): ReactElement {
               'flex flex-1 flex-col laptop:mr-8 laptop:max-w-[27.5rem]',
             )}
           >
-            <OnboardingTitleGradient className="mb-4 typo-large-title tablet:typo-mega1">
-              Where developers suffer together
-            </OnboardingTitleGradient>
-
-            <h2 className="mb-8 typo-body tablet:typo-title2">
-              We know how hard it is to be a developer. It doesn&apos;t have to
-              be. Personalized news feed, dev community and search, much better
-              than what&apos;s out there. Maybe ;)
-            </h2>
-
+            <OnboardingHeadline />
```

Comment:
Maybe to avoid the ! in the banner we're adding, we can actually accept the specific classnames for the title from both places.

Reply:
You mean like adding a `typoClassName`?

Reply:
No I mean not having default typo setup in the classnames in OnboardingHeadline, but actually accepting the typos from classname props here just like we do on the other place we use OnboardingHeadline
`ex. className={{ title: '!typo-mega3', description: '!typo-title3' }}` If we do the same here we avoid the !

Reply:
Ah, yep. That makes sense. Let's do that 🚀